packages/dm-core/src/components/TreeView.tsx

```diff
@@ -53,6 +56,14 @@ const GetIcon = (props: { node: TreeNode; expanded: boolean }) => {
     return <label>{'{}'}</label>
   }
   switch (node.type) {
+    case EBlueprint.FILE: {
+      const fileColor = '#2965FF'
```

Comment:
is `#2965FF`a color from EDS?

Reply:
nope, but need to cleanup the tree later on, think this is okey for now